widgetssdk/src/main/java/com/glia/widgets/view/unifiedui/theme/base/BadgeTheme.kt

```diff
@@ -0,0 +1,8 @@
+package com.glia.widgets.view.unifiedui.theme.base
+
+internal data class BadgeTheme(
+    val textColor: ColorTheme?,
+    val background: LayerTheme?,
+    val textSize: Float?, // Size in SP
+    val textStyle: Int?, //Typeface.NORMAL
```

Comment:
Minor comment. To keep it perfectly nice, let's add a space after `//` as you did in the line above.